R/utils.R

```diff
@@ -31,8 +31,16 @@ get_example_data <- function(which_data) {
       full.names = TRUE
     )
     return(readr::read_rds(file))
+  } else if (which_data == "annotation") {
+    file <- list.files(
+      pattern = "pgPEN_annotations.txt",
+      recursive = TRUE,
+      system.file("extdata", package = "gimap"),
+      full.names = TRUE
+    )
+    return(readr::read_tsv(file, show_col_types = FALSE))
   } else {
-    stop("Specification for `which_data` not understood; Need to use 'gimap', count', or 'meta'")
+    stop("Specification for `which_data` not understood; Need to use 'gimap', count', 'annotation', 'crtl_genes', or 'meta'")
```

Comment:
Are we including a control genes annotation here that can be loaded from example data or just doing the read from DepMap as done above?

Reply:
Ah this is a remnant from how I wrote this originally before I changed it. Good catch. I'll change it.